src/Components/FormComboBox.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { comboBox } from '../../node_modules/uswds/src/js/components';
+
+import { ValidationState } from '../Models/PatientFieldData';
+import ValidationMessage from './ValidationMessage';
+
+function FormComboBox({ label, property, required, onChange, options, showRequiredHint, size, validationState, value }) {
+  const ref = useRef();
+  const [focused, setFocused] = useState(false);
+  const [customOption, setCustomOption] = useState(null);
```

Comment:
I've added some additional state to support allowing the user to input a new value that is not in the options list, that I call the "customOption". This did not seem to be a part of the default USWDS comboBox functionality.